src/App/App.tsx

```diff
@@ -1,39 +1,45 @@
 import React, { useEffect, useState } from 'react';
 import './App.css';
 import Welcome from "../Welcome/Welcome";
-import { Route, Switch } from "react-router-dom";
+import { Route, Switch, Link } from "react-router-dom";
 import Quotes from '../Quotes/Quotes';
 import Form from '../Form/Form';
+import Favorites from '../Favorites/Favorites';
 
 export interface Quote {
   q: string;
   a: string;
-  h: string;
+  h?: string;
 }
 
 const App = () => {
   const [allQuotes, setAllQuotes] = useState<Quote[]>([])
+	const [favList, setFavList] = useState<Quote[]>([])
 
   return (
+		<div>
+			<nav>
+				<Link to='/Favorites'>Favorites</Link>
```

Comment:
Nice addition on the Links! This makes me remember we still have to wrap the button on the Welcome page to link to home!